### ISSUE TITLE
[retina-net]How to set learning rate and strategies when using sync bn?

### ISSUE THREAD

**Issue by YilanWang** [open-mmlab/mmdetection#1629]
When I want to use sync bn, I find that the learning rate will become a difficult parameter to adjust. It is difficult to use the previous default learning rate setting.
For eg, when I use retina-net to train my own face datasets and do face detection tast, it's ok to use default learning strategies.  Once I add sync bn, I can only set 2 the batch size to 4 (2 imgs_per_gpu and 2 gpus), otherwise, the result is too bad.  
What advice and strategies are there for setting and adjusting the learning rate when using sync bn? thanks!

**Comment by hellock** [comment]
We just adopt the same setting (2 img/gpu * 8gpu) w/ or w/o SyncBN. On COCO it is okay.

**Comment by YilanWang** [comment]
> We just adopt the same setting (2 img/gpu * 8gpu) w/ or w/o SyncBN. On COCO it is okay.

thank you for reply! If I want to use a larger batch size, such as (2img*32gpus or more), what experience and advice do you have on this scale? What should I pay attention to in the setting of the learning rate? many thanks for your reply!

**Comment by hellock** [comment]
SyncBN does not affects the learning rate and we follow the linear scaling rule when we use 32 or more GPUs. You may try longer warmup iters and set the warmup ratio smaller.
